hicexplorer/hicAggregateContacts.py

```diff
@@ -71,11 +87,26 @@ def parse_arguments(args=None):
                            choices=['total-counts', 'z-score', 'obs/exp', 'none'],
                            default='none')
 
-    parserOpt.add_argument('--avgType',
-                           help='Type of average used in the output matrix. Options are mean and median. Default is median.',
-                           choices=['mean', 'median'],
+    parserOpt.add_argument('--operationType',
+                           help='Type of the operation to be applied to the output matrix. Options are mean and median. Default is median.',
+                           choices=['sum', 'mean', 'median'],
                            default='median')
 
+    parserOpt.add_argument('--perChr',
+                           help='if set, it generates a plot per chromosome. It is only affected if '
+                           'intera-chromosomal contacts are of interest.',
+                           action='store_true',
+                           required=False)
+
+    parserOpt.add_argument('--largeRegionsOperation',
+                           help='If a given coordinate in the bed file is larger than '
+                           'a bin of the inpt matrix, by default only the first bin is taken into account. '
+                           'However there are more posibilities to handel such a case. User can '
+                           'ask for the last bin, sum of the bins, mean or median of the bins which cover '
+                           'this region.',
+                           choices=['first', 'last', 'sum', 'mean', 'median'],
+                           default='first')
+
```

Comment:
It is not that clear that it influences also the start range upstream or downstream of the bins.